bin/pycbc_compress_bank

```diff
@@ -146,23 +178,40 @@ for ii in range(imin, imax):
     htilde = bank[ii-imin]
     tmplt = bank.table[ii-imin]
     kmax = htilde.end_idx
-    
-    # get the compressed sample points
-    if args.compression_algorithm == 'mchirp':
-        sample_points = compress.mchirp_compression(tmplt.mass1, tmplt.mass2,
-            fmin, (kmax-1)*df, min_seglen=args.t_pad, df_multiple=df).astype(
-            real_same_precision_as(htilde))
-    elif args.compression_algorithm == 'spa':
-        sample_points = compress.spa_compression(htilde, fmin, (kmax-1)*df,
-            min_seglen=args.t_pad).astype(real_same_precision_as(htilde))
+    tmplt_approximant = htilde.approximant
+    if num_compression_algorithms == 2:
+        if tmplt_approximant == approximant1:
+            compression_type = compression_type1
+        if tmplt_approximant == approximant2:
+            compression_type = compression_type2
```

Comment:
Having created the approximant name -> compression algorithm dictionary above, here you could get the `compression_type` by just passing this template's approximant to the dictionary.